ui/src/layout/Findings/Secrets/TabSecretDetails.jsx

```diff
@@ -22,7 +22,7 @@ const TabSecretDetails = ({data}) => {
                     <TitleValueDisplayRow>
                         <TitleValueDisplay title="File path">{filePath}</TitleValueDisplay>
                     </TitleValueDisplayRow>
-                    <FindingsDetailsCommonFields foundOn={foundOn} invalidatedOn={invalidatedOn} />
+                    <FindingsDetailsCommonFields firstSeen={firstSeen} lastSeen={lastSeen} />
```

Comment:
I'm not very great at UI but maybe the `LatSeenBy` should also be passed here?

Reply:
https://github.com/openclarity/vmclarity/pull/1758#discussion_r1629625393